test/transcoding/global_block.cl

```diff
@@ -4,18 +4,8 @@
 // removed
 
 // RUN: %clang_cc1 -O0 -triple spir-unknown-unknown -cl-std=CL2.0 -x cl %s -emit-llvm-bc -o %t.bc
-// TODO: currently max version is limited to 1.1 for this test. Issues here
```

Comment:
an we modify this test in a way that it will check both cases when max version is limited to 1.1 and when it is not limited at all?